frontend/src/pages/GamePage.tsx

```diff
@@ -1,25 +1,26 @@
-import React, { useState } from "react";
+// import { quizQuestions } from "./questions";
+
+import React, { useEffect, useState } from "react";
 import Terminal, { ColorMode, TerminalOutput } from "react-terminal-ui";
-import { quizQuestions } from "./questions";
+import axios from "axios";
 
 const GamePage: React.FC = () => {
-    const [terminalLineData, setTerminalLineData] = useState([
+    const [terminalLineData, setTerminalLineData] = useState<React.ReactNode[]>([
         <TerminalOutput>Welcome to the Life Choices Game!</TerminalOutput>,
```

Comment:
See the `/config/index.ts` file, use the name defined there.
That is,
instead of `Welcome to Life Choices Game!`
write: `Welcome to {config.name}`